python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py

```diff
@@ -136,8 +136,10 @@ def optimize(self, model: Model,
                   If x is a dataset, y will be ignored (since targets will be obtained from x).
         :param validation_data: (optional) An unbatched tf.data.Dataset object for accuracy
                evaluation. This is only needed when users care about the possible accuracy drop.
-        :param input_spec: A (tuple or list of) ``tf.TensorSpec`` defining the
-                           shape/dtype of the input.
+        :param input_spec: (optional) A (tuple or list of) ``tf.TensorSpec``
+                           defining the shape/dtype of the input. This is only required when
+                           you have a custom Keras model(no input/output layer is explicitly
```

Comment:
A space seems to be missing between before the "("

Reply:
done